salesforce-composite/2.8/modules/ROOT/pages/salesforce-composite-connector-examples.adoc

```diff
@@ -1,8 +1,8 @@
-= Salesforce Composite - Examples - Mule 4
+= Salesforce Composite Connector - Examples - Mule 4
 
 == SObject Tree Operation Example
 
-This example uses the following operations to connect to Salesforce, create and execute an SObject tree operation, and transform and output the results of the executed operation in JSON format:
+This example uses the following operations to connect to Salesforce, create and execute an SObject tree, and transform and output the results in JSON format:
```

Comment:
You aren't executing the sobject tree, you are executing the operation on the sobject tree
So maybe just:
...create an SObject tree to connect to Salesforce and transform and output the results in JSON format.